internal/trivy/scan_job.go

```diff
@@ -193,7 +193,9 @@ func (f *filesystemScanJobBuilder) container(spec stasv1alpha1.ContainerImageSca
 	container.Args = []string{
 		string(f.TrivyCommand),
 		"--skip-files",
-		FsScanTrivyBinaryPath,
+		// FIXME: Somehow Trivy strips off the first part of the path
+		"run/image-scanner/trivy",
```

Comment:
```suggestion
		"**run/image-scanner/trivy",
```

This also works, and might make the solution more future-proof for eventual future fixes by Aqua?

Reply:
Could also ignore both, so that it works when the issue is resolved
```suggestion
		"run/image-scanner/trivy",
		"--skip-files",
		"/var/run/image-scanner/trivy",
```